Tribler/Core/Config/config.spec

```diff
@@ -4,7 +4,6 @@ state_dir = string(default='')
 ec_keypair_filename = string(default='')
 megacache = boolean(default=True)
 videoanalyserpath = string(default='')
-minport = integer(default=7760)
```

Comment:
This was once the minimum port at which libtorrent starts searching for an open port. Finding new ports is currently done by libtorrent, outside of Tribler.